src/dialog/snippets/dialog.html

```diff
@@ -0,0 +1,37 @@
+<style>
+  .demo-dialog .mdl-dialog {
+    width: 1024px;
+  }
+</style>
+
+<div class="mdl-dialog mdl-transition-in" aria-label="Mango (Fruit)">
+  <div class="mdl-dialog-content" style="max-width:800px;max-height:810px; ">
+    <div>
+      <p>
+        The mango is a juicy stone fruit belonging to the genus Mangifera, consisting of numerous tropical fruiting trees, cultivated mostly for edible fruit. The majority of these species are found in nature as wild mangoes. They all belong to the flowering plant family Anacardiaceae. The mango is native to South and Southeast Asia, from where it has been distributed worldwide to become one of the most cultivated fruits in the tropics.
+      </p>
+
+      <img style="margin: auto; max-width: 100%;" alt="Lush mango tree" src="img/mangues.jpg">
```

Comment:
Is it showing correctly? If not, you can remove it. We don't need images in the dialog yet.